powerfactory_tools/exporter/exporter.py

```diff
@@ -2263,7 +2261,7 @@ def calc_load_mv_power_asym(self, load: PFTypes.LoadMV) -> LoadMV:
         # in PF for consumer: ind. cosphi = under excited; cap. cosphi = over excited
         cosphi_dir_cons = CosphiDir.UE if load.pf_recap == 0 else CosphiDir.OE
         # in PF for producer: ind. cosphi = over excited; cap. cosphi = under excited
-        cosphi_dir_prod = CosphiDir.OE if load.pfg_recap == 0 else CosphiDir.UE
+        cosphi_dir_prod = CosphiDir.UE if load.pfg_recap == 1 else CosphiDir.OE
```

Comment:
isn't this the same logic as before?

Reply:
```suggestion
        cosphi_dir_prod = CosphiDir.UE if load.pfg_recap else CosphiDir.OE
```

Reply:
yes, this was not a fix , just a design change as check/assigment is at all places performed like this